requirements.txt

```diff
@@ -3,5 +3,5 @@ exceptiongroup==1.1.0
 iniconfig==2.0.0
 packaging==23.0
 pluggy==1.0.0
-pytest==7.2.1
+pytest==7.3.2
 tomli==2.0.1
```

Comment:
since we are using poetry we have to add this dependencies to poetry, also we have to add `freegun`, please read the documentation of poetry and add by running `poetry add ...`

Reply:
Please delete this file as well

Reply:
I add freezgun to poetry, I have to add all the dependencies of requirements.txt ?